test/controllers/IllnessStartDateController.test.ts

```diff
@@ -0,0 +1,208 @@
+import { expect } from 'chai';
+import { MOVED_TEMPORARILY, OK, UNPROCESSABLE_ENTITY } from 'http-status-codes';
+import { after, before } from 'mocha';
+import request from 'supertest';
+
+import 'app/controllers/EvidenceDownloadController';
+import { Appeal } from 'app/models/Appeal';
+import {
+    ENTRY_PAGE_URI,
+    ILLNESS_START_DATE_PAGE_URI
+} from 'app/utils/Paths';
+
+import { createApp } from 'test/ApplicationFactory';
+import { createDefaultAppeal } from 'test/models/AppDataFactory';
+
+const pageHeading: string = 'When did the illness start?';
+const errorSummaryHeading: string = 'There is a problem';
+const invalidStartDayErrorMessage: string = 'You must enter a day';
+const invalidStartMonthErrorMessage: string = 'You must enter a month';
+const invalidStartYearErrorMessage: string = 'You must enter a year';
+const invalidDateErrorMessage: string = 'Enter a real date';
+const invalidDateFutureErrorMessage: string = 'Start date must be today or in the past';
+let initialIllnessReasonFeatureFlag: string | undefined;
+
+describe('IllnessStartDateController', () => {
+
+    before(() => {
+        initialIllnessReasonFeatureFlag = process.env.ILLNESS_REASON_FEATURE_ENABLED;
+    });
+
+    after(() => {
+        process.env.ILLNESS_REASON_FEATURE_ENABLED = initialIllnessReasonFeatureFlag;
+    });
+
+    const appeal = createDefaultAppeal();
+
+    describe('GET request', () => {
+
+        it('should return 200 when trying to access the page', async () => {
+
+            process.env.ILLNESS_REASON_FEATURE_ENABLED = '1';
+
+            const app = createApp({appeal});
+            await request(app).get(ILLNESS_START_DATE_PAGE_URI).expect(response => {
+                expect(response.status).to.be.equal(OK);
+                expect(response.text).to.contain(pageHeading);
+            });
+        });
+
+        it('should return 200 when trying to access page with a session', async () => {
+
+            process.env.ILLNESS_REASON_FEATURE_ENABLED = '1';
+
+            const appealWithReasons = {
+                penaltyIdentifier: {
+                    companyNumber: 'NI000000',
+                    penaltyReference: 'A00000001'
+                },
+                reasons: {
+                    other: {
```

Comment:
That should not be in the tests. We do not care about other reasons, lets presume users just started with illness reason.